### ISSUE TITLE
Examples should not rely on default axes

### ISSUE THREAD

**Issue by VisualMelon** [oxyplot/oxyplot#1547]
Some examples in the example library depend on the default axes (e.g. LineSeries > MarkerTypes). #992 says we want to remove default axes for 3.0. This requires adding axes to any examples that need them.

This should be as simple as pasting the following into the appropriate methods:

    model.Axes.Add(new LinearAxis { Position = AxisPosition.Bottom });
    model.Axes.Add(new LinearAxis { Position = AxisPosition.Left });

It would not make sense to do this automatically in the example library somehow, because some examples (e.g. those showing rendering capabilities) should not have axes at all.

Related: #1544

**Comment by Jonarw** [comment]
Maybe an extension method `WithDefaultAxes(this PlotModel model)` would be nice to have in the ExampleLibrary? Or maybe even in Core library?

**Comment by VisualMelon** [comment]
I'd personally be hesitant to add such a method for a couple of reasons:
 1. We probably don't want to encourage throwing unlabelled axes at the screen
 2. It's not obvious what it should do if the plot already has axes
 3. That sort of name doesn't communicate anything about what the axes actually are (i.e. linear/whatever)

I should probably sneakily update the OP so that it doesn't sound like I'm advocate for the opposites of points 1 and 3 in the example library ;)

I'd be more open to methods which set up a whole plot model to some extent with names like "Linear" that take titles as parameters; however, I've often written these in the past and always seem to end up regretting it. If they did reduce the barrier to entry for the library (e.g. if making a pair of axes with titles was one like of code instead of 3) they might have a place, but I don't think I wouldn't myself push for these sorts of helpers in the main library.

**Comment by Jonarw** [comment]
Yes, I agree it's probably not a good idea for the core library.

However in the ExampleLibrary I could see the use even for some non-issue examples. Often the axes are just not that important for demonstrating a particular thing and could be distracting if they have to be explicitly created every time.

**Comment by VisualMelon** [comment]
You've convinced me: a throw-away method in the example library is a good idea.

I'll be happy to go through and fix all the examples so I can push the transpose fix through (all the erroring plots in the exporter tests are bugging me) if no one beats me to it.
